scripts/generate-javadoc-modules.groovy

```diff
@@ -0,0 +1,30 @@
+import groovy.json.*;
+
+// define sort order for plugins
+def keyComparator = [compare: { e1, e2 -> e1.name.compareToIgnoreCase(e2.name) }] as Comparator
+
+// TODO: automatic module discovery from WAR?
+def modules = new ModuleList()
```

Comment:
It will still require a rework, because module discovery requires version anyway. The autodiscovery discovery would need to generate the ModuleList, not just a map